### PR TITLE
ICP-8045 Add join name translations for china devices

### PR DESCRIPTION
**Pull request by FenMei007S**
@tpmanley  Dear Tom, plz review it.

### CODE REVIEW

File: devicetypes/smartthings/smartsense-motion-sensor.src/smartsense-motion-sensor.groovy [hunk 1]
```diff
@@ -39,7 +39,7 @@ metadata {
 		fingerprint inClusters: "0000,0001,0003,000F,0020,0402,0500", outClusters: "0019", manufacturer: "SmartThings", model: "motionv5", deviceJoinName: "Motion Sensor"
 		fingerprint inClusters: "0000,0001,0003,0020,0400,0500,0B05", outClusters: "0019", manufacturer: "Bosch", model: "RFPR-ZB", deviceJoinName: "Bosch Motion Sensor"
 		fingerprint inClusters: "0000,0001,0003,000F,0020,0402,0500", outClusters: "0019", manufacturer: "Bosch", model: "RFDL-ZB-MS", deviceJoinName: "Bosch Motion Sensor"
-		fingerprint inClusters: "0000,0001,0003,0020,0402,0500", outClusters: "0019", manufacturer: "Samjin", model: "motion", deviceJoinName: "Motion Sensor"
+		fingerprint inClusters: "0000,0001,0003,0020,0402,0500", outClusters: "0019", manufacturer: "Samjin", model: "motion", deviceJoinName: "Samjin Motion Sensor"
```
Comment (tpmanley) [inline devicetypes/smartthings/smartsense-motion-sensor.src/smartsense-motion-sensor.groovy]: These join names have to stay the way they are

File: devicetypes/smartthings/smartsense-motion-sensor.src/i18n/messages.properties [hunk 2]
```diff
@@ -31,3 +31,6 @@
 '''{{ device.displayName }} battery has too much power: (> 3.5) volts.'''.ko={{ device.displayName }} 배터리 전력이 너무 높습니다(3.5볼트 초과).
 '''{{ device.displayName }} battery was {{ value }}%'''.ko={{ device.displayName }}의 남은 배터리 {{ value }}%
 #==============================================================================
+
+# Chinese
+'''Samjin Motion Sensor'''.zh-cn=人体传感器 Samsung Connect (IM6001-MTP)
```
Comment (tpmanley) [inline devicetypes/smartthings/smartsense-motion-sensor.src/i18n/messages.properties]: The join name has to stay just `Motion Sensor` and then changed here as well

File: devicetypes/smartthings/smartsense-multi-sensor.src/smartsense-multi-sensor.groovy [hunk 3]
```diff
@@ -34,7 +34,7 @@ metadata {
 		fingerprint inClusters: "0000,0001,0003,0402,0500,0020,0B05,FC02", outClusters: "0019", manufacturer: "CentraLite", model: "3321"
 		fingerprint inClusters: "0000,0001,0003,0402,0500,0020,0B05,FC02", outClusters: "0019", manufacturer: "CentraLite", model: "3321-S", deviceJoinName: "Multipurpose Sensor"
 		fingerprint inClusters: "0000,0001,0003,000F,0020,0402,0500,FC02", outClusters: "0019", manufacturer: "SmartThings", model: "multiv4", deviceJoinName: "Multipurpose Sensor"
-		fingerprint inClusters: "0000,0001,0003,0020,0402,0500,FC02", outClusters: "0019", manufacturer: "Samjin", model: "multi", deviceJoinName: "Multipurpose Sensor"
+		fingerprint inClusters: "0000,0001,0003,0020,0402,0500,FC02", outClusters: "0019", manufacturer: "Samjin", model: "multi", deviceJoinName: "Samjin Multipurpose Sensor"
```
Comment (tpmanley) [inline devicetypes/smartthings/smartsense-multi-sensor.src/smartsense-multi-sensor.groovy]: Keep the original join name here too

**Comment by tpmanley** [comment]
Need to do the door lock too

**Comment by FenMei007S** [comment]
@tpmanley Tom,updated, plz review it again

**Review by tpmanley — APPROVED** [review]

**Review by greens — APPROVED** [review]

**Pull request merged**